packages/neuron-wallet/src/block-sync-renderer/sync/indexer-connector.ts

```diff
@@ -173,9 +177,13 @@ export default class IndexerConnector {
     for await (const cell of collector.collect()) {
       //somehow the lumos indexer returns an invalid hash type "lock" for hash type "data"
       //for now we have to fix it here
-      const cellOutput: any = cell.cell_output
-      if (cellOutput.type?.hash_type === 'lock') {
-        cellOutput.type.hash_type = 'data'
+      const cellOutput = cell.cellOutput
+      // FIXME
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-expect-error
+      if (cellOutput.type?.hashType === 'lock') {
```

Comment:
Has Lumos fixed `returns an invalid hash type "lock" for hash type "data"`? If so, we can remove this magic.

Reply:
I'm not sure, so I just left the patch here, and added some logs to help to locate the bugs